### PR TITLE
Bridge client

### LINKED ISSUE #60: Bridge client - deployment
# Bounty

## Scope

- A deploy function on the bridge client
- Some way to tun it
-

## Deliverables

- Code
-

## Funding Circle

Nervos

## Bounty Owner/Gardener

<@your github username> as **<Role>**
@TheReturnOfJan as CKBDeveloper

## Roles
bounty gardener: <@your github username> / X DAI 
bounty worker: @TheReturnOfJan  / 80%
bounty reviewer: name / share

## Gardener checklist

- [x] I have attached one of the `size-` labels to this bounty
- [ ] I have submitted this bounty via http://bounty.leapdao.org/viewform
- [ ] I have added this bounty to bounty board here https://github.com/orgs/leapdao/projects/6

### LINKED ISSUE #61: Bridge client - deposit
# Bounty

## Scope

- A deposit function on the bridge client
- Some way to run it
-

## Deliverables

- Code
-

## Funding Circle

Nervos

## Bounty Owner/Gardener

<@your github username> as **<Role>**
@TheReturnOfJan as CKBDeveloper

## Roles
bounty gardener: <@your github username> / X DAI 
bounty worker: @TheReturnOfJan  / 80%
bounty reviewer: name / share

## Gardener checklist

- [x] I have attached one of the `size-` labels to this bounty
- [ ] I have submitted this bounty via http://bounty.leapdao.org/viewform
- [ ] I have added this bounty to bounty board here https://github.com/orgs/leapdao/projects/6

### PR DESCRIPTION
**Pull request by TheReturnOfJan**
Resolves #60 and Resolves #61

### CODE REVIEW

File: packages/client/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    // "preserveSymlinks": true,              /* Do not resolve the real path of symlinks. */
+    // "allowUmdGlobalAccess": true,          /* Allow accessing UMD globals from modules. */
+
+    /* Source Map Options */
+    // "sourceRoot": "",                      /* Specify the location where debugger should locate TypeScript files instead of source locations. */
+    // "mapRoot": "",                         /* Specify the location where debugger should locate map files instead of generated locations. */
+    // "inlineSourceMap": true,               /* Emit a single file with source maps instead of having a separate file. */
+    // "inlineSources": true,                 /* Emit the source alongside the sourcemaps within a single file; requires '--inlineSourceMap' or '--sourceMap' to be set. */
+
+    /* Experimental Options */
```
Comment (mwaa) [inline packages/client/tsconfig.json:59]: I think we should remove all the commented code make it lean.

File: packages/client/src/test.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      index: "0x0",
+    },
+    dep_type: "code",
+  },
+  ANYONE_CAN_PAY_SCRIPT: {
+    code_hash: "0xe683b04139344768348499c23eb1326d5a52d6db006c0d2fece00a831f3660d7",
+    hash_type: "type",
+    args: "0x",
+  },
+  DEPOSIT_CODE_HASH: "0xd7aa21f5395d0bb03935e88ccd46fd34bd97e1a09944cec3fcb59c340deba6cf",
```
Comment (mwaa) [inline packages/client/src/test.ts:32]: Are this values for the Testnet? 
Can we have a deployment comment that mentions all the hashes that need to be replaced in this file?

File: packages/client/tsconfig.json (hunk 1 above)
Comment (TheReturnOfJan) [reply packages/client/tsconfig.json:59]: I like having all the options in case we need them in the future.

**Review by mwaa — APPROVED** [review]
Got to test transaction. Deposit is recorded successfully.

**Pull request merged**